index.html

```diff
@@ -129,1517 +70,43 @@
-                        provided in the introduction, explaining how to get access to the documentation.
-                    </li>
-                </ul>
-            </section>
-        </section>
-    </section>
-
-    <section id="binding-examples" class="informative">
-        <h2>Examples of Thing Descriptions with Protocol Binding Templates</h2>
-
-        <p>
-            The following TD examples uses HTTP, CoAP and MQTT Protocol Binding Templates.
-            These TDs have Context Extensions which assume that there is a CoAP and MQTT in RDF vocabulary similar to
-            [[?HTTP-in-RDF10]] that is accessible via the namespaces <code>http://www.example.org/coap-binding#</code>
-            and <code>http://www.example.org/mqtt-binding#</code>, respectively.
-            Please note that the TD context at <code>"https://www.w3.org/2019/wot/td/v1"</code> already includes the [[?HTTP-in-RDF10]],
-            so HTTP context extensions can be directly used and make sure to look at the Binding Template <a>subspecifications</a> to find the 
-            most up-to-date vocabulary terms and examples.
-        </p>
-        <p>
-            The context extensions we see below have the following instructions to the TD Consumer:
-        </p>
         <ul>
             <li>
-                <code>"htv:methodName"</code> member instructs the Consumer which HTTP method has to be applied 
-                (e.g., <code>"GET"</code> for retrieving a resource or <code>"POST"</code> for sending data to a resource).
+                If you are interested in the binding mechanism, please refer to the <a
+                href="https://w3c.github.io/wot-thing-description/#bindings">Bindings section of the WoT Thing Description
+                document</a>.
             </li>
             <li>
-                <code>"cov:method"</code> member instructs the Consumer which CoAP method has to be
-                applied (e.g., <code>"GET"</code> for the CoAP Method Code 0.01, <code>"POST"</code> for the CoAP Method Code
-                0.02, or <code>iPATCH</code> for CoAP Method Code 0.07).
+                If you want to find the list of the available bindings, the <a href="https://w3c.github.io/wot-bindings-registry/">WoT Bindings Registry</a> is where they will be located.
+                Until they are registered in the registry, they are listed in the <a href="https://github.com/w3c/wot-binding-templates#publications">Readme</a>.
             </li>
             <li>
-                <code>"mqv:controlPacket"</code> member instructs the Consumer which MQTT command has
-                to be applied (e.g., <code>"subscribe"</code> for the subscribing to a topic and <code>"unsubscribe"</code> for
-                unsubscribing).
+                The latest version of this document is available as a <a href="https://www.w3.org/TR/wot-binding-templates/">group note</a>.
```

Comment:
```suggestion
                The latest version of this document is available as a <a href="https://www.w3.org/TR/2024/NOTE-wot-binding-templates-20240528/">group note</a>.
```

I think it should use the dated-URI since otherwise THIS document will be shown again when clicking on the link in the future...

Reply:
Good catch, committing :)